vagrant/provisioning/roles/httpd/templates/ldap-auth-restricted-linuxadmins-helper.script2

```diff
@@ -0,0 +1,14 @@
+AuthType Basic
+AuthName "Restricted applications"
+AuthBasicProvider ldap
+RewriteEngine On
+#LDAPReferrals Off
+# Perform LDAP lookup for user to check group membership
+AuthLDAPUrl "{{ ldap_url }}/DC=appdev,DC=armedia,DC=com?sAMAccountName?sub?(objectClass=*)"
+AuthLDAPBindDN "{{ ldap_base }}"
+AuthLDAPBindPassword "{{ ldap_bind_password }}"
+AuthzLDAPAuthoritative off
+#AuthLDAPInitialBindAsUser off
+AuthLDAPRemoteUserAttribute sAMAccountName
+Require ldap-filter &(objectClass=person)(objectClass=user)(memberOf=CN=linuxadmins,"{{ ldap_base }}")((userAccountControl:1.2.840.113556.1.4.803:=2))
```

Comment:
The linux admin CN should also be configured in the facts file

Reply:
Also samAccountName should be a fact... on the happy day we switch from ActiveDirectory to FreeIPA, the user id will be uid, not samAccountName.